code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -95,10 +95,9 @@
 	cover_range = WEED_RANGE_CORE
 	node_type = /obj/effect/alien/weeds/node/pylon/core
 	var/hardcore = FALSE
-
 	var/next_attacked_message = 5 SECONDS
 	var/last_attacked_message = 0
-
+	var/warn = FALSE
```

Comment:
what's the point of this var? It only warns you the first time someone tries to destroy the hive core, so they can do it silently if they attempt it once then cancel.

Reply:
will make warn reset to FALSE after 45 seconds and make NO keep warn to false